test_unstructured/partition/test_org.py

```diff
@@ -15,3 +15,18 @@ def test_partition_org_from_file(filename="example-docs/README.org"):
 
     assert elements[0] == Title("Example Docs")
     assert elements[0].metadata.filetype == "text/org"
+
+
+def test_partition_org_from_filename_exclude_metadata(filename="example-docs/README.org"):
+    elements = partition_org(filename=filename, include_metadata=False)
+
+    for i in range(len(elements)):
+        assert any(elements[i].metadata.to_dict()) is False
```

Comment:
This should be an empty dictionary, asserting that it's equal to `{}` might have been better, but since this pattern is already added everywhere I say keep it.

Reply:
I can do a quick search for that assert and change them to dicts